### DISCUSSION TITLE
How to increase the connection limit of websocket

### DISCUSSION THREAD

**Discussion by lightyear67** [cesanta/mongoose#2539] [Ask questions, get answers] [answered]
Hello everyone.
I am having a problem where I have created a server but only have a maximum of 64 connections. When I try a new connection again, it cannot connect successfully. How should I increase the connection limit? Thanks.
This is a part of my code sample, did I make a mistake somewhere.

![图片](https://github.com/cesanta/mongoose/assets/45167452/46230e07-ca30-4e5b-8f46-7df293a0421e)
![图片](https://github.com/cesanta/mongoose/assets/45167452/bd148d7f-8404-48b9-8aa8-e7a99ce1ecc8)
The “count” can only be up to 64.

**Comment by scaprile** [comment, marked as answer]
Please:
- Do not paste pictures, use github permalinks or markdown or attach network capture files
- Search for already asked (and answered) similar issues/questions before opening a new one
- Know your development environment and its limits

There are no limits in Mongoose. You are using Windows and are using select()
https://github.com/cesanta/mongoose/issues/1962#issuecomment-1396318383
https://github.com/cesanta/mongoose/issues/1962#issuecomment-1396732667
#2368 
#2326
